src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -372,12 +371,13 @@ private static ThreeState TryGetBestResult(ArrayBuilder<MemberResolutionResult<T
             MemberResolutionResult<TMember> firstSupported = default(MemberResolutionResult<TMember>);
             MemberResolutionResult<TMember> firstUnsupported = default(MemberResolutionResult<TMember>);
 
-            var supportedInPriorityOrder = new MemberResolutionResult<TMember>[5]; // from highest to lowest priority
-            const int requiredParameterMissingPriority = 0;
-            const int nameUsedForPositionalPriority = 1;
-            const int noCorrespondingNamedParameterPriority = 2;
-            const int noCorrespondingParameterPriority = 3;
-            const int badNonTrailingNamedArgument = 4;
+            var supportedInPriorityOrder = new MemberResolutionResult<TMember>[6]; // from highest to lowest priority
+            const int duplicateNamedArguments = 0;
+            const int requiredParameterMissingPriority = 1;
+            const int nameUsedForPositionalPriority = 2;
+            const int noCorrespondingNamedParameterPriority = 3;
+            const int noCorrespondingParameterPriority = 4;
+            const int badNonTrailingNamedArgument = 5;
```

Comment:
> const int badNonTrailingNamedArgument = 5; [](start = 11, length = 43)

Curious: why do we use constants here to establish priority vs. giving the memwber of MemberResolutionKind an explicit numeric value and using that as priority?

Reply:
> Curious: why do we use constants here to establish priority vs. giving the memwber of MemberResolutionKind an explicit numeric value and using that as priority?

There are way more MemberResolutionKinds than priorities. This is the only place where we prioritize things. And we avoid spooky action at a distance. :-)